### ISSUE TITLE
Inconsistency ... in consistency measures?

### ISSUE THREAD

**Issue by lcottret** [opencobra/memote#714]
Hi,

First, thanks for this very useful tool !
I have a problem with our last reconstruction: we have a score of 0.0% in "Stoichiometric consistency". In our SBML model, each species has a fbc:chemicalFormula and a fbc:charge attributes and we are pretty sure that main of the reactions are balanced. Furthemore, with a score of 0%, I would expect that the number of unconserved metabolites equals to the total number of metabolites, what is not the case. Is there a bug or something that I don't understand in this score ?

Regards,
Ludo Cottret

**Comment by carrascomj** [comment]
I think you are talking about this one https://github.com/opencobra/memote/issues/708#issuecomment-760097229

TL;DR that measure was changed to a binary score, where it is 0 if it the matrix is inconsistent and 100% otherwise.

**Comment by lcottret** [comment]
Ok, thanks ! This is quite stringent since it completely forbids some extra generic reactions, often useful in modelling...

**Comment by Midnighter** [comment]
What kind of reactions are you talking about @lcottret? Exchange, sink, demand, and biomass reactions (as far as detected by memote) are automatically ignored for this analysis.

**Comment by carrascomj** [comment]
Also, remember that the information about inconsistencies is still computed and shown in the report if you want to check it!

**Comment by famosab** [comment]
I had this issue as well. Thanks for the explanation! When do you plan to update the web version of memote to version 0.12.0?

**Comment by Midnighter** [comment]
> I had this issue as well. Thanks for the explanation! When do you plan to update the web version of memote to version 0.12.0?

Good point. I will update it in the next two weeks.
